bundles/org.openhab.binding.etherrain/src/main/resources/ESH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,153 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
This XML file should be formatted with tabs, but given the alignment looks good, its not breaking for me, but better auto-format with Eclipse to correct it.